test/units/modules/storage/netapp/test_na_ontap_gather_facts.py

```diff
@@ -119,7 +119,7 @@ def test_ensure_command_called(self, mock_ems_log):
         my_obj = my_module(module)
         my_obj.server = MockONTAPConnection('vserver', 'SVMadmin')
         with pytest.raises(KeyError) as exc:
-            my_obj.get_all()
+            my_obj.get_all('all')
```

Comment:
it should be 
my_obj.get_all(['all'])

Reply:
@lonico ,

Thanks for spotting this. I fixed it, and I also added some checks for some of the subset stuff.